pkg/javadb/client.go

```diff
@@ -82,8 +83,12 @@ func (u *Updater) Update() error {
 }
 
 func Init(cacheDir, javaDBRepository string, skip, quiet bool, registryOption ftypes.RegistryOptions) {
+	// Add the schema version as a tag if the tag doesn't exist.
```

Comment:
ditto

Reply:
Added comment in https://github.com/aquasecurity/trivy/pull/6219/commits/91bcb0ab8ce79d6e58ec07600ee10fd2287fe4f7